conformance/patches/find_provs.patch

```diff
@@ -0,0 +1,23 @@
+--- node_modules/interface-ipfs-core/src/dht/find-provs.js
++++ node_modules/interface-ipfs-core/src/dht/find-provs.js
+@@ -38,12 +38,13 @@ module.exports = (common, options) => {
+     before('add providers for the same cid', async function () {
+       this.timeout(10 * 1000)
+
+-      const cids = await Promise.all([
+-        nodeB.object.new('unixfs-dir'),
+-        nodeC.object.new('unixfs-dir')
++      const blob = Buffer.from('blorb')
++      const blocks = await Promise.all([
++        nodeB.block.put(blob),
++        nodeC.block.put(blob)
+       ])
+
+-      providedCid = cids[0]
++      providedCid = blocks[0].cid.toString()
```

Comment:
This however might be a problematic change, at least the string conversion part. I think the conversion should be handled by js-ipfs-http-client.

```suggestion
-      providedCid = cids[0]
+      providedCid = blocks[0].cid
```

Reply:
didn't think we cared, as I consider this to just be duct-tape to make our tests pass (while preserving the actual thing being tested), but I guess I can change this at `js-ipfs-http-client` level

Reply:
Does the conversion fail without the `.toString()` removed?

Reply:
it actually doesn't - maybe it was something I experimented with initially and just left it there; regardless, `should be able to find providers` is commented out now so it's not needed in this PR